src/commands/cmd_set.cc

```diff
@@ -283,6 +285,55 @@ class CommandSInter : public Commander {
   }
 };
 
+/*
+ * description:
+ *
+ *    syntax: SINTERCARD numkeys key [key ...] [LIMIT limit]
+ *
+ *    params: the checked cardinality is unlimited (limit <= 0).
```

Comment:
Redis prohibits the negative `LIMIT` parameter:

```
127.0.0.1:6379> sintercard 2 set1 set2 limit -5
(error) ERR LIMIT can't be negative
127.0.0.1:6379> 
```

BTW: right now the phrase `the checked cardinality is unlimited` sounds weird to me. 
Since `Kvrocks` supports `Redis` protocol, if you want to add some documentation, you can just copy-paste sentences directly from `Redis` docs.

Reply:
Thanks, I got it.